src/test/kotlin/org/rust/ide/annotator/fixes/ReplaceWithStdMemDropFixTest.kt

```diff
@@ -0,0 +1,213 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import org.rust.ProjectDescriptor
+import org.rust.WithStdlibAndDependencyRustProjectDescriptor
+import org.rust.ide.annotator.RsAnnotatorTestBase
+import org.rust.ide.annotator.RsErrorAnnotator
+
+// The Drop trait (which must be recognized for this to work) isn't recognized without stdlib.
+@ProjectDescriptor(WithStdlibAndDependencyRustProjectDescriptor::class)
```

Comment:
You don't need this descriptor here because it adds some additional dependency crates besides stdlib.
Use `WithStdlibRustProjectDescriptor` if you want to add only stdlib or (that even better) you can mock `Drop` trait here and don't add stdlib in these tests at all.
Just add
```rust
#[lang = "drop"]
pub trait Drop { 
    fn drop(&mut self); 
}
```
to each test case. Note, it will work only if you apply suggestions about qualified name checking in the inspection